docs/source/usage/plugins/sliceEmittance.rst

```diff
@@ -7,6 +7,7 @@ The plugin computes the total emittance and the slice emittance (for ten combine
 
 Currently, it outputs only the emittance of the transverse momentum space x-px.
 
+More details on the implementation and tests can be found in the master thesis [Rudat2019]_. 
```

Comment:
Not sure, but shouldn't it be `Master`, not `master`?

Reply:
I just checked wikipedia: https://en.wikipedia.org/wiki/Thesis#Germany (I did not know it either.)
It should neither be `master` nor `Master`, it should be `master's` - I will fix it. 😕